File: integration-tests/hibernate-reactive-panache/src/main/java/io/quarkus/it/panache/reactive/CatDto.java
package io.quarkus.it.panache.reactive;

import io.quarkus.hibernate.reactive.panache.common.ProjectedFieldName;
import io.quarkus.runtime.annotations.RegisterForReflection;

@RegisterForReflection
public class CatDto {

    public String name;

    public String ownerName;

    public CatDto(String name, @ProjectedFieldName("owner.name") String ownerName) {
        this.name = name;
        this.ownerName = ownerName;
    }

}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/CatDto.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.common.ProjectedFieldName
import io.quarkus.runtime.annotations.RegisterForReflection

@RegisterForReflection
class CatDto(var name: String, @param:ProjectedFieldName("owner.name") var ownerName: String)
